src/Tests/TestExtensions.cs

```diff
@@ -41,7 +41,7 @@ await engine
             var client = await (await engine).Engine.GetEngineService<IAzureClient>();
             if (client is AzureClient azureClient && azureClient.ActiveWorkspace is MockAzureWorkspace workspace)
             {
-                workspace.AddProviders("ionq", "honeywell");
+                workspace.AddProviders("ionq", "quantinuum");
```

Comment:
should we keep `honeywell` scenario here to make sure it continues to work?

Reply:
Fixed.